image_proc/package.xml

```diff
@@ -11,7 +11,7 @@
   <maintainer email="mfergs7@gmail.com">Michael Ferguson</maintainer>
 
   <license>BSD</license>
-  <url type="website">https://index.ros.org/p/image_proc/github-ros-perception-image_pipeline/</url>
+  <url type="website">https://index.ros.org/p/image_proc/</url>
```

Comment:
what about this link https://docs.ros.org/en/rolling/p/image_proc/ ?

Reply:
I think linking to the package page on index.ros.org is more helpful, because you can get to everything from there, e.g., API docs, repository/code.